.github/workflows/build_wheels.yml

```
@@ -78,6 +80,8 @@ jobs:
           python-version: 3.12
         if: runner.os == 'Windows'
       - run: bash ./ci/cibw_before_all_windows.sh "${{ github.workspace }}"
+        env:
+          ARCH: ${{ runner.arch }}
```

Not sure it makes a difference (if it does, please explain), but this would keep the style consistent across the file
```suggestion
          ARCH: ${{ matrix.arch }}
```

---

@neutrinoceros this is looks meaningful! thanks